auth-server/test/services/email/email.spec.ts

```diff
@@ -0,0 +1,132 @@
+import 'mocha'
+import { expect } from 'chai'
+import rewiremock from 'rewiremock'
+
+import EmailVerificationToken from '../../../src/model/EmailVerificationToken'
+import PasswordResetToken from '../../../src/model/PasswordResetToken'
+import User from '../../../src/model/User'
+
+const noop = () => {}
+
+describe('Email Service', () => {
+	const email = 'test@email.com'
+	const password = 'testpass'
+	const username = 'testuser'
+	const name = 'test name'
+	const salt = 'testsalt'
+	let apiKey = ''
+
+	before(() => {
+		apiKey = process.env.SENDGRID_API_KEY
```

Comment:
Do we need this if we later have `setApiKey: noop`?

Reply:
nope, removing

Reply:
done